### ISSUE TITLE
Need to collapse down 'asset' data

### ISSUE THREAD

**Issue by gdbassett** [vz-risk/veris#173]
There are 4 enumerations in the 'asset' section that are largely duplicative: Cloud, Governance, Hosting, Management, and Ownership.  'Cloud' is irregularly used (mostly to answer 'NA' or 'Unknown'), of the other 4, they are primarily used to note that the asset was the victim's in the victim's data center.  

Recommend collapsing 'cloud' and 'hosting' (though maintaining a way of specifying 'cloud' since it is of significant interest to many people).  Recommend collapsing 'governance' and 'management' to a single feature (since the distinction isn't clear).  Recommend leaving ownership.  Recommend clarifying all descriptions to make the differences between the enumerations abundantly clear.

```
        "cloud": {
          "enum": [
            "NA", 
            "No", 
            "Hosting error", 
            "User breakout", 
            "Unknown", 
            "Customer attack", 
            "Hosting governance", 
            "Hypervisor", 
            "Other", 
            "Partner application"
          ], 
          "type": "string"
        }, 
        "governance": {
          "items": {
            "enum": [
              "3rd party hosted", 
              "Unknown", 
              "3rd party managed", 
              "3rd party owned", 
              "Other", 
              "Victim governed", 
              "Personally owned", 
              "Internally isolated"
            ], 
            "type": "string"
          }, 
          "minItems": 1, 
          "type": "array", 
          "uniqueItems": true
        }, 
        "hosting": {
          "enum": [
            "External shared", 
            "External dedicated", 
            "NA", 
            "Internal", 
            "External", 
            "Unknown", 
            "Other"
          ], 
          "type": "string"
        }, 
        "management": {
          "enum": [
            "NA", 
            "Internal", 
            "Other", 
            "External", 
            "Unknown"
          ], 
          "type": "string"
        }, 
        "ownership": {
          "enum": [
            "Customer", 
            "Unknown", 
            "Other", 
            "Victim", 
            "NA", 
            "Employee", 
            "Partner"
          ], 
          "type": "string"
        }, 
```

**Comment by gdbassett** [comment]
Additionally consider removing `asset.accessibility` as it is rarely filled in.

**Comment by gdbassett** [comment]
Hosting tells you how an asset is hosted.  Cloud tells you if the cloud had an affect on security.  hosting.3rd party = "hosted in cloud"?

Marc will look at it and provide a from->to w/ definitions.

**Comment by gdbassett** [comment]
Include cleaning up 'accessability'.  (Consider adding "Partner": "Accessible from partner system" to asset.accessibility per issue #139)

    "accessibility": {
      "External": "Publicly accessible",
      "Internal": "Internally accessible",
      "Isolated": "Internally isolated or restricted environment",
      "NA": "Not applicable",
      "Other": "Accessibility known but not listed",
      "Unknown": "Unknown"
    },

**Comment by gdbassett** [comment]
Addressed in commit f566e3b.  Removed accessibility. removed governance and mapped some of it to hosting/management/ownership.  Added definitions for all hosting/management/ownership.  Most per @Spitler.

**Comment by gdbassett** [comment]
Marc completed this.  This year we are finishing the  cloud portion.
